CHANGELOG.md

```diff
@@ -23,6 +24,167 @@
+  entities specified in the given `kong.yml` file in bulk
+  [#4284](https://github.com/Kong/kong/pull/4284)
+- New command: `kong config init` to generate a template `kong.yml`
+  file to get you started
+- New command: `kong config parse kong.yml` to verify the syntax of
+  the `kong.yml` file before using it
+- New option `--wait` in `kong quit` to ease graceful termination when using orchestration tools
+
+##### Admin API
+
+- New Admin API endpoint: `/config` to replace the configuration of
+  Kong entities entirely, replacing it with the contents of a new
+  declarative config file
+  - When using the new `database=off` configuration option,
+    the Admin API endpoints for entities (such as `/routes` and
+    `/services`) are read-only, since the configuration can only
+    be updated via `/config`
+    [#4308](https://github.com/Kong/kong/pull/4308)
+- Admin API endpoints now support searching by tag
+  (for example, `/consumers?tags=example_tag`)
+  - You can search by multiple tags:
+     - `/services?tags=serv1,mobile` to search for services matching tags `serv1` and `mobile`
+     - `/services?tags=serv1/serv2` to search for services matching tags `serv1` or `serv2`
+- New Admin API endpoint `/tags/` for listing entities by tag: `/tags/example_tag`
+
+##### PDK
+
+- New PDK function: `kong.client.get_protocol` for obtaining the protocol
+  in use during the current request
+  [#4307](https://github.com/Kong/kong/pull/4307)
+- New PDK function: `kong.nginx.get_subsystem`, so plugins can detect whether
+  they are running on the HTTP or Stream subsystem
+  [#4358](https://github.com/Kong/kong/pull/4358)
+
+##### Plugins
+
+- :fireworks: Support for ACL **authenticated groups**, so that authentication plugins
+  that use a 3rd party (other than Kong) to store credentials can benefit
+  from using a central ACL plugin to do authorization for them
+  [#4013](https://github.com/Kong/kong/pull/4013)
+- The Kubernetes Sidecar Injection plugin is now bundled into Kong for a smoother K8s experience
+  [#4304](https://github.com/Kong/kong/pull/4304)
+- aws-lambda: includes AWS China region.
+  Thanks [@wubins](https://github.com/wubins) for the patch!
+  [#4176](https://github.com/Kong/kong/pull/4176)
+
+### Changes
+
+##### Dependencies
+
+- The required OpenResty version is still 1.13.6.2, but for a full feature set
+  including stream routing and Service Mesh abilities with mutual TLS, Kong's
+  [openresty-patches](https://github.com/kong/openresty-patches) must be
+  applied (those patches are already bundled with our official distribution
+  packages). The openresty-patches bundled was updated in Kong 1.1.0 to include
+  the `stream_realip_module` as well.
+  [#4163](https://github.com/Kong/kong/pull/4163)
+  Kong in HTTP(S) Gateway scenarios does not require these patches.
+- Service Mesh abilities require at least OpenSSL version 1.1.1. In our
+  official distribution packages, OpenSSL has been bumped to 1.1.1b.
```

Comment:
OpenSSL hasn't really been bumped. #4440 should take care of it (assuming we will release from kong-build-tools as planned).

Reply:
> assuming we will release from kong-build-tools as planned

Yes, we will, and we will also build the old-name packages with kong-distributions. @hutchic, the kong-distributions builds for 1.1.0 are using OpenSSL 1.1.1b, right?

Reply:
Yes we are going to release with kong-build-tools and kong-distributions. The kong-distributions 1.1.0 branch has been pinned to `OpenSSL 1.1.1b`

Doing a quick check I think we have missed making the change on the kong@next `.requirements` file ( https://github.com/Kong/kong/blob/next/.requirements#L4 ) which is what tells kong-build-tools the version to use (possible I missed this change or didn't communicate how to configure the change)